packages/frontend/src/lib/Card.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+export let title: string | undefined = undefined;
+export let description: string | undefined = undefined;
+
+export let href: string | undefined = undefined;
```

Comment:
suggestion: use svelte5 runes

```suggestion
interface Props {
title: string;
description: string;
href: string
}

let {title, description, href} : Props = $props();
```